src/xdocs/config_modifier.xml

```diff
@@ -22,6 +22,206 @@
+          <tr>
+            <td>violateImpliedStaticField</td>
+            <td>
+              Control whether to enforce that <code>static</code> is not being implied by the
+              compiler on interface fields (true).</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>true</code></td>
+            <td>8.11</td>
+          </tr>
+          <tr>
+            <td>violateImpliedFinalField</td>
+            <td>
+              Control whether to enforce that <code>final</code> is not being implied by the
+              compiler on interface fields (true).</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>true</code></td>
+            <td>8.11</td>
+          </tr>
+          <tr>
+            <td>violateImpliedPublicMethod</td>
+            <td>
+              Control whether to enforce that <code>public</code> is not being implied by the
+              compiler on interface methods (true).</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>true</code></td>
+            <td>8.11</td>
+          </tr>
+          <tr>
+            <td>violateImpliedAbstractMethod</td>
+            <td>
+              Control whether to enforce that <code>abstract</code> is not being implied by the
+              compiler on interface methods (true).</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>true</code></td>
+            <td>8.11</td>
+          </tr>
+          <tr>
+            <td>violateImpliedPublicNested</td>
+            <td>
+              Control whether to enforce that <code>public</code> is not being implied by the
+              compiler on interface nested types (true).</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>true</code></td>
+            <td>8.11</td>
+          </tr>
+          <tr>
+            <td>violateImpliedStaticNested</td>
+            <td>
+              Control whether to enforce that <code>static</code> is not being implied by the
+              compiler on interface nested types (true).</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>true</code></td>
+            <td>8.11</td>
+          </tr>
+        </table>
+      </subsection>
+
+      <subsection name="Examples">
+        <p> To configure the check: </p>
+        <source>
+&lt;module name=&quot;InterfaceMemberImpliedModifier&quot;/&gt;
+        </source>
+
+        <p>
+          To configure the check to check methods and fields but not nested types:
+        </p>
+        <source>
+&lt;module name=&quot;InterfaceMemberImpliedModifier&quot;&gt;
+  &lt;property name=&quot;violateImpliedPublicNested&quot; value=&quot;false&quot;/&gt;
+  &lt;property name=&quot;violateImpliedStaticNested&quot; value=&quot;false&quot;/&gt;
+&lt;/module&gt;
+        </source>
```

Comment:
please add here small snipped of code to show what behavior will be (with trailing comments "//violation").

Unfortunately not that much good cases in our documentation to show you example, but user constantly complain about this. We try at least in new Checks enforce this good practice.

Good example: http://checkstyle.sourceforge.net/config_annotation.html#AnnotationLocation , there is human description, config, code example.

Reply:
Done